schema/partner_show.js

```diff
@@ -64,6 +65,11 @@ const PartnerShowType = new GraphQLObjectType({
     press_release: markdown(),
     start_at: date,
     end_at: date,
+    exhibition_period: {
```

Comment:
In Eigen we called this [ausstellungsdauer](https://github.com/artsy/eigen/blob/123764e3eeb8f2f3067453955e9da6b07fed7fbe/Artsy/Categories/Apple/NSDate%2BDateRange.m#L19-L21), but I’ve noticed that that was not something that was shared across clients, e.g. Force calls it ‘running dates’.

I went with the translation of ‘ausstellungsdauer’, but I’m open to a naming contest.

/cc @orta